src/utils.js

```diff
@@ -133,7 +135,12 @@ function clone(object) {
   const newObject = create(null);
 
   for (const [property, value] of entries(object)) {
-    if (getOwnPropertyDescriptor(object, property) !== undefined) {
+    const isPropertyExist = Object.prototype.hasOwnProperty.call(
+      object,
+      property
+    );
```

Comment:
JavaScript does not protect the property name `hasOwnProperty`, so objects with properties with this name may return incorrect results.

This is a more stable way to use it.

#### Related
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#using_hasownproperty_as_a_property_name)
[eslint no-prototype-builtins](https://eslint.org/docs/latest/rules/no-prototype-builtins)